_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title:  "Introduction to Mixing Audio"
+author: flavius
+categories: [ audio ]
+image: ""
+featured: false
+---
+What is audio mixing? Is it helpful or a hindrance? What are my options for mixing audio?
+How much does it cost?
+
+### - Introduction
+
+Audio mixing, what is it? Audio Mixing is where you take multiple input sources of audio and put them into a single or fewer number output signals. An example of input sources would be a microphone, game audio, stream alert audio. An example of an output signal would be our headphones, speakers, or stream. Can you achieve audio mixing for streams and videos without a digital or physical audio mixer? Mostly, but in practice it is significantly more complicated to route audio properly without using some form of audio mixing software or hardware when dealing with multiple sources.
+
```

Comment:
Maybe insert a section here or after "Goals of Audio Mixing" about OBS audio mixing and why it's not sufficient?

"### Can't I 'mix' audio with OBS?"

Reply:
Covered in audio mixing options